astropy/visualization/stretch.py

```diff
@@ -47,38 +47,36 @@ def _prepare(values, out=None, clip=True):
 @six.add_metaclass(InheritDocstrings)
 class BaseStretch(BaseTransform):
     """
-    Base class for the stretch classes, which, when called with an array of
-    values in the range [0:1], return an transformed array of values, also in
-    the range [0:1].
+    Base class for the stretch classes, which, when called with an array
+    of values in the range [0:1], return an transformed array of values,
+    also in the range [0:1].
     """
 
-    def __call__(self, values, out=None, clip=True):
+    def __call__(self, values, clip=True, out=None):
```

Comment:
I suspect that not many people were actually calling stretch instances directly on the data (with or without the `out` keyword), but I'm happy to mention it in the changelog.  @astrofrog?